tests/unit/llms/test_llm_loader.py

```diff
@@ -36,25 +36,9 @@ def test_constructor_unknown_provider():
 
 def test_constructor_unsupported_provider():
     """Test how a configured but an unsupported provider (not openai, bam, etc) is checked for."""
-    test_provider = "test-provider"
-    test_provider_type = "unsupported-provider-type"
-    config.config = Config(
-        {
-            "llm_providers": [
-                {
-                    "name": test_provider,
-                    "type": "bam",
-                    "models": [
-                        {
-                            "name": constants.GRANITE_13B_CHAT_V1,
-                        }
-                    ],
-                }
-            ]
-        }
-    )
-    config.config.llm_providers.providers[test_provider].type = test_provider_type
-    config.llm_config = config.config.llm_providers
+    providers = LLMProviders([{"name": "some-provider", "models": [{"name": "model"}]}])
+    config.init_empty_config()
+    config.config.llm_providers = providers
 
     with pytest.raises(UnsupportedProviderError):
         LLMLoader(provider=test_provider, model=constants.GRANITE_13B_CHAT_V1)
```

Comment:
```suggestion
    load_llm(provider=test_provider, model=constants.GRANITE_13B_CHAT_V1)
```

?

Reply:
a few other spots too

Reply:
Yup, I should have mentioned that pushed just the code for approval (without tests update).